nxc/protocols/smb.py

```diff
@@ -583,22 +583,23 @@ def create_smbv3_conn(self):
             return False
         return True
 
-    def create_conn_obj(self, no_smbv1=False):
+    def create_conn_obj(self):
```

Comment:
why not keeping the no_smbv1 and add another line after ?

`no_smbv1 = self.args.force_smbv2 if self.args.force_smbv2 else no_smbv1`

this way the logic after is not touched

Reply:
Let see, since we have force-smbv2, so can we add like `self.force_smbv2 = self.args.force_smbv2`?

Reply:
I think it's much better than before now